docs/configuration/recording_rules.md

```diff
@@ -45,7 +45,8 @@ dashboards, which need to query the same expression repeatedly every time they
 refresh.
 
 Recording and alerting rules exist in a rule group. Rules within a group are
-run sequentially at a regular interval.
+run sequentially at a regular interval. The names of recording and alerting rules
+must be [valid metric names](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
```

Comment:
We usually do /docs/...  for URLs like this

Reply:
👋 

Thank for the review. I took a look through the repo and I didn't see any links in the style you mentioned:
```
~/projects/prometheus-ahayworth-fork ahayworth-docs-update-rule-names 7s
❯ git grep -ni 'docs' docs  | egrep -v '(microsoft|amazon|google|mesosphere)\.com'
docs/configuration/alerting_rules.md:46:templates](https://prometheus.io/docs/visualization/consoles).  The `$labels`
docs/configuration/alerting_rules.md:100:[Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) takes on this
docs/configuration/configuration.md:12:instances](https://prometheus.io/docs/concepts/jobs_instances/), as well as
docs/configuration/configuration.md:623:[integrations](https://prometheus.io/docs/operating/integrations/#file-service-discovery) with this
docs/configuration/configuration.md:762:* `__meta_kubernetes_pod_container_init`: `true` if the container is an [InitContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
docs/configuration/configuration.md:769:  in the [lifecycle](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase).
docs/configuration/configuration.md:1322:[integrations](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage)
docs/configuration/configuration.md:1368:[integrations](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage)
docs/configuration/recording_rules.md:49:must be [valid metric names](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
docs/configuration/template_examples.md:57:formatting of results, and linking to the [expression browser](https://prometheus.io/docs/visualization/browser/).
docs/configuration/template_reference.md:75:| graphLink  | expr | string | Returns path to graph view in the [expression browser](https://prometheus.io/docs/visualization/browser/) for the expression. |
docs/configuration/template_reference.md:76:| tableLink  | expr | string | Returns path to tabular ("Console") view in the [expression browser](https://prometheus.io/docs/visualization/browser/) for the expression. |
docs/index.md:10:[prometheus.io](https://prometheus.io/docs/prometheus/latest/).
docs/installation.md:31:[Data Volume Container](https://docs.docker.com/engine/admin/volumes/volumes/)
docs/migration.md:187:See [Configure a Security Context for a Pod or Container](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
docs/querying/functions.md:144:[histogram](https://prometheus.io/docs/concepts/metric_types/#histogram). (See
docs/querying/functions.md:145:[histograms and summaries](https://prometheus.io/docs/practices/histograms) for
docs/querying/functions.md:150:The [histogram metric type](https://prometheus.io/docs/concepts/metric_types/#histogram)
docs/storage.md:46:Note that a limitation of the local storage is that it is not clustered or replicated. Thus, it is not arbitrarily scalable or durable in the face of disk or node outages and should be treated as you would any other kind of single node database. Using RAID for disk availiablity, [snapshots](https://prometheus.io/docs/prometheus/latest/querying/api/#snapshot) for backups, capacity planning, etc, is recommended for improved durability. With proper storage durability and planning storing years of data in the local storage is possible.
docs/storage.md:48:Alternatively, external storage may be used via the [remote read/write APIs](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage). Careful evaluation is required for these systems as they vary greatly in durability, performance, and efficiency.
docs/storage.md:50:For further details on file format, see [TSDB format](https://github.com/prometheus/prometheus/blob/master/tsdb/docs/format/README.md).
docs/storage.md:105:To learn more about existing integrations with remote storage systems, see the [Integrations documentation](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).

~/projects/prometheus-ahayworth-fork ahayworth-docs-update-rule-names
```

The link in question is actually to the [docs repo](https://github.com/prometheus/docs) and it wasn't obvious to me if I could make a relative link in _this_ repo and have it resolve correctly when built.

However, if you assert that it will, I will be happy to change it.

Reply:
Ah sorry, I was looking in the wrong repo.